std/StandardToken.sol

```diff
@@ -53,7 +53,7 @@ contract StandardToken is Token {
 		return true;
```

Comment:
Why is this function not caught by it? It has `success`, but is never set. Same goes for `doTransfer`.